spec/support/guide/rails_5_controller_compatibility.rb

```diff
@@ -0,0 +1,11 @@
+if Rails.version >= '5'
+  module Rails5ControllerCompatibility
+    def get(action, params = {})
+      super action, params: params
+    end
+  end
```

Comment:
This is interesting. Might end up using this in the marketplace app instead of changing every controller spec! Although, I imagine we'll need to do the change at some point